bevrand.frontend/assets/js/login.js

```diff
@@ -35,7 +35,7 @@ $("#loginForm").submit(function( event ) {
 function loginUser(userList) {
     $.ajax({
         type: "POST",
-        url: `${config.proxyHostname}/api/login`,
+        url: `${config.proxyHostname}/authentication-api/login`,
```

Comment:
Bij login moeten we even bedenken of we wel willen dat users met email en username inloggen ipv alleen email.

We hebben een endpoint om op basis van email de username te achterhalen:

url: `${config.proxyHostname}/playlist-api/Users?by-email=${email}`

Die zouden we kunnen gebruiken om de username op te halen en dan naar de proxy te sturen, dat is dan helaas wel een call extra. Op langere termijn willen we dit wellicht oplossen aan de authenticatie api kant.